driver-core/src/main/com/mongodb/ReadPreferenceHedgeOptions.java

```diff
@@ -0,0 +1,120 @@
+/*
+ * Copyright 2008-present MongoDB, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.mongodb;
+
+import com.mongodb.annotations.Immutable;
+import org.bson.BsonBoolean;
+import org.bson.BsonDocument;
+
+/**
+ * Options to apply to hedged reads in the server.
+ *
+ * @since 4.1
+ * @mongodb.server.release 4.4
+ */
+@Immutable
+public final class ReadPreferenceHedgeOptions {
+    private final Boolean hedgedReads;
```

Comment:
This should be of type primitive `boolean`.  If it were ever actually set to null, there are multiple place that would throw exceptions.  But it never is, so no point in making it type `Boolean`